pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -333,3 +333,31 @@ func addStandardNodeAttributes(attr map[string]*mpb.Attributes_AttributeValue, p
 		}
 	}
 }
+
+func serviceConfig(serviceHostname string, dest *model.ServiceInstance, config model.IstioConfigStore, disableCheck, disableReport bool) *mccpb.ServiceConfig {
+	sc := v1.ServiceConfig(serviceHostname, dest, config, disableCheck, disableReport)
+	addDestinationServiceAttributes(sc.MixerAttributes.Attributes, serviceHostname)
+	return sc
+}
+
+func addDestinationServiceAttributes(attrs map[string]*mpb.Attributes_AttributeValue, destinationHostname string) {
+	svcName, svcNamespace := nameAndNamespace(destinationHostname)
+	attrs[v1.AttrDestinationService] = attrStringValue(destinationHostname)
```

Comment:
I'm in favor of using literal strings for attributes. It makes it easy to search and find all occurrences.